tsconfig.json

```diff
@@ -13,7 +13,10 @@
     "noImplicitThis": true,
     "moduleResolution": "node",
     "module": "commonjs",
-    "baseUrl": "."
+    "baseUrl": ".",
+    "paths": {
+      "react": ["node_modules/@types/react"]
```

Comment:
this fixes the issue with linking packages (`yarn link`) having the same type definitions